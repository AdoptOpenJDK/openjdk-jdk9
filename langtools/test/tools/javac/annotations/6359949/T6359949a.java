/*
 * @test    /nodynamiccopyright/
 * @bug     6359949
 * @summary (at)Override of static shouldn't be accepted (compiler shouldissue an error/warning)
 * @compile/fail/ref=T6359949a.out -XDrawDiagnostics  T6359949a.java
 */

class Example {
    public static void example() {

    }
}

class Test extends Example {
    @Override
    public static void example() {

    }
}
