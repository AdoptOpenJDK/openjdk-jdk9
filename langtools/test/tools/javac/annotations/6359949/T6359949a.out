T6359949a.java:15:5: compiler.err.method.does.not.override.superclass
1 error
